prime-angular-frontend/src/app/modules/enrolment/pages/job/job.component.ts

```diff
@@ -242,7 +212,9 @@ export class JobComponent extends BaseEnrolmentProfilePage implements OnInit, On
     this.oboSites.clear();
```

Comment:
Got `Cannot read property 'clear' of null ` error on regulatory page when hitting 'Save and continue" button.

Testing case: new enrolment, selected more than 1 HA in care setting,
![image](https://user-images.githubusercontent.com/58282608/113038798-c5151500-914b-11eb-9524-447c71fb3106.png)